verification/cbmc/proofs/aws_cryptosdk_private_commitment_eq/aws_cryptosdk_private_commitment_eq_harness.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+// #include <aws/cryptosdk/cipher.h>
+#include <aws/cryptosdk/private/cipher.h>
+#include <make_common_data_structures.h>
+
+void aws_cryptosdk_private_commitment_eq_harness() {
+    /* Nondet Input */
+    struct aws_byte_buf *buf1 = malloc(sizeof(*buf1));
+    struct aws_byte_buf *buf2 = malloc(sizeof(*buf2));
```

Comment:
Could you also create another issue on AWS C Common to update `ensure_byte_buf_has_allocated_buffer_member` to include this allocation? We have updated this style since then.